ketcher-autotests/README.md

```diff
@@ -89,22 +89,23 @@ Docker runs automatically in the pipeline after pushing changes to the repositor
 
 #### If you want to update snapshots based on the rc version:
 
-1. Directory "ketcher-autotests": Set .env variable KETCHER_URL=https://rc.test.lifescience.opensource.epam.com
+1. Directory "ketcher-autotests": Set .env variable KETCHER_URL=link_to_rc
 2. Directory "ketcher": Build autotests for docker:`npm run docker:build:autotests`
```

Comment:
I am assuming this means "example" directory? Since we dont have one called "ketcher"

Reply:
[EDIT] oh, I see that this means project rot directory